app/controllers/katello/concerns/hosts_controller_extensions.rb

```diff
@@ -18,6 +18,17 @@ def action_permission
       included do
         prepend Overrides
 
+        def update_multiple_taxonomies(type)
+          registered_host = @hosts.detect { |host| host.subscription_facet }
+          unless registered_host.nil?
+            error "Unregister host #{registered_host.name} before assigning an organization"
```

Comment:
Formatting strings in this way make translations more challenging. It should be: `_("Unregister host %s before assigning an Organization") % registered_host.name`

Reply:
oh right, I'll update!